src/components/form-components/Section.tsx

```diff
@@ -51,10 +51,14 @@ const Section: React.FC<Props> = (props) => {
       <Heading level={3} margin="none">
         {translateCopy(question.sections.name)}
       </Heading>
+      {question.sections.instructions && (
+        <Markdown margin={{ top: 'small' }} size="small">
+          {translateCopy(question.sections.instructions)}
```

Comment:
Finally, render the markdown-type instructions property using our Markdown form component. Only render if it exists.